service/storage_service.cc

```diff
@@ -937,6 +938,12 @@ void storage_service::bootstrap() {
 
     _gossiper.check_seen_seeds();
 
+    _db.invoke_on_all([this] (database& db) {
+        for (auto& cf : db.get_non_system_column_families()) {
+            cf->notify_stream_recv_start();
+        }
```

Comment:
Do we really need this? I was under the impression that storage_service::_is_bootstrap_mode could be used to determine when bootstrap phase is done.

Reply:
We do, because the places in which we check don't have access to the storage service.

Reply:
Understood. thanks